### PR TITLE
Added example and temporary search queries history

### PR DESCRIPTION
**Pull request by AS2**

### CODE REVIEW

File: src/MangaJet/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -16,6 +16,7 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.MangaJet"
         android:usesCleartextTraffic="true">
+
```
Comment (Sairsey) [inline src/MangaJet/app/src/main/AndroidManifest.xml:19]: Remove it

File: src/MangaJet/app/src/main/java/com/mangajet/mangajet/ui/search/SearchViewModel.kt [hunk 2]
```diff
@@ -173,6 +179,39 @@ class SearchViewModel : ViewModel() {
...
 
+    fun setSomeRandomPopularHint(binding: SearchFragmentBinding)  {
+        hintJob?.cancel()
+        hintJob = viewModelScope.launch(Dispatchers.IO) {
+            // choose random resource
+            var arrayOfResources = arrayListOf<Int>()
+            for (i in Librarian.LibraryName.values().indices)
+                if (Librarian.settings.CHOSEN_RESOURCES[i])
+                    arrayOfResources.add(i)
+            val randomResource = /*(arrayOfResources.indices).random()*/ 0
```
Comment (Sairsey) [inline src/MangaJet/app/src/main/java/com/mangajet/mangajet/ui/search/SearchViewModel.kt:190]: Set random please

File: src/MangaJet/app/src/main/java/com/mangajet/mangajet/ui/search/SearchViewModel.kt [hunk 3]
```diff
@@ -173,6 +179,39 @@ class SearchViewModel : ViewModel() {
...
+
+            // choose random manga
+            val popularMangas =
+                Librarian.getLibrary(Librarian.LibraryName.values()[randomResource])!!.
+                getPopularManga(1 + 1 + 1 + 1 + 1)
```
Comment (Sairsey) [inline src/MangaJet/app/src/main/java/com/mangajet/mangajet/ui/search/SearchViewModel.kt:195]: To constant

**Review by Sairsey — CHANGES_REQUESTED** [review]

File: src/MangaJet/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (AS2) [reply src/MangaJet/app/src/main/AndroidManifest.xml:19]: NO.

File: src/MangaJet/app/src/main/java/com/mangajet/mangajet/ui/search/SearchViewModel.kt (hunk 2 above)
Comment (AS2) [reply src/MangaJet/app/src/main/java/com/mangajet/mangajet/ui/search/SearchViewModel.kt:190]: HO-HO-HO.

File: src/MangaJet/app/src/main/java/com/mangajet/mangajet/ui/search/SearchViewModel.kt (hunk 3 above)
Comment (AS2) [reply src/MangaJet/app/src/main/java/com/mangajet/mangajet/ui/search/SearchViewModel.kt:195]: YEEES.

**Review by Sairsey — APPROVED** [review]

**Pull request merged**